app/assets/stylesheets/geo.scss

```diff
@@ -67,3 +67,92 @@
     // scss-lint:enable SelectorDepth
   }
 }
+
+.opacity-control {
+  background-color: #a9acb1;
+  border-radius: 15px;
+  color: black;
+  font: bold 18px 'Lucida Console', Monaco, monospace;
+  display: block;
+  height: 200px;
+  left: 11px;
+  position: relative;
+  top: 15px;
+  width: 5px;
+
+  .opacity-handle {
+    background-color: #fff;
+    border-radius: 4px;
+    border: 1px solid #eee;
+    cursor: ns-resize;
+    font-size: 10px;
+    height: 26px;
+    left: -11px;
+    line-height: 26px;
+    position: absolute;
+    text-align: center;
+    top: 0;
+    width: 26px;
+    z-index: 20;
+    box-shadow: 0 1px 5px rgba(0,0,0,0.65);
+
+    &:hover {
+      background-color: #f4f4f4;
```

Comment:
Can you use `var(--stanford-fog-light)` here instead? You might need to add this to https://github.com/sul-dlss/sul-embed/blob/main/app/assets/stylesheets/palette.scss

Reply:
Done. I also updated it everywhere we used the color (1 other place).